tests/test_plugin.py

```diff
@@ -117,10 +117,38 @@ def test_icon_emoji():
     with mock.patch('requests.post') as mock_post:
         testdir.runpytest('--slack_channel', slack_hook_channel,
                           '--slack_hook', slack_hook_host,
-                          '--slack_success_icon', slack_hook_icon_emoji_success,
-                          '--slack_failed_icon', slack_hook_icon_emoji_failed)
+                          '--slack_success_emoji', slack_hook_icon_emoji_success,
+                          '--slack_failed_emoji', slack_hook_icon_emoji_failed)
 
         called_data = json.loads(mock_post.call_args[1]['data'])
         emoji = called_data['icon_emoji']
 
         assert emoji == expected_emoji
+
+
+@pytest.mark.parametrize('test_input,expected_url', [
+    ('1 == 1', ':sunny:'),
+    ('2 == 1', ':rain_cloud:'),
+])
+def test_pytest_slack_custom_icons(testdir, test_input, expected_url):
```

Comment:
Please add the test that will check that emoji  will be overridden by url option.

Reply:
Agreed. Fixed.